edit/buildozer.go

```diff
@@ -454,6 +455,85 @@ func cmdCopyNoOverwrite(opts *Options, env CmdEnvironment) (*build.File, error)
 	return copyAttributeBetweenRules(env, attrName, from)
 }
 
+// cmdDictAdd adds a key to a dict, if that key does _not_ exit already.
+func cmdDictAdd(opts *Options, env CmdEnvironment) (*build.File, error) {
+	attr := env.Args[0]
+	args := env.Args[1:]
+
+	dict := &build.DictExpr{}
+	currDict, ok := env.Rule.Attr(attr).(*build.DictExpr)
+	if ok {
+		dict = currDict
+	}
+
+	for _, x := range args {
+		kv := strings.Split(x, ":")
+		expr := build.StringExpr{
+			Value:       kv[1],
+			TripleQuote: false,
+			Token:       "",
+		}
+
+		prev := DictionaryGet(dict, kv[0])
+		if prev == nil {
+			// Only set the value if the value is currently unset.
+			DictionarySet(dict, kv[0], &expr)
+		}
+		env.Rule.SetAttr(attr, dict)
+	}
+	return env.File, nil
+}
+
+// cmdDictSet adds a key to a dict, overwriting any previous values.
+func cmdDictSet(opts *Options, env CmdEnvironment) (*build.File, error) {
+	attr := env.Args[0]
+	args := env.Args[1:]
+
+	dict := &build.DictExpr{}
+	currDict, ok := env.Rule.Attr(attr).(*build.DictExpr)
+	if ok {
+		dict = currDict
+	}
+
+	for _, x := range args {
+		kv := strings.Split(x, ":")
+		expr := build.StringExpr{
+			Value:       kv[1],
+			TripleQuote: false,
+			Token:       "",
+		}
+		// Set overwrites previous values.
+		DictionarySet(dict, kv[0], &expr)
+		env.Rule.SetAttr(attr, dict)
+	}
+	return env.File, nil
+}
+
+// cmdDictRemove removes a key from a dict.
+func cmdDictRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
+	attr := env.Args[0]
+	args := env.Args[1:]
+
+	thing := env.Rule.Attr(attr)
+	dictAttr, ok := thing.(*build.DictExpr)
+	if ok {
```

Comment:
if !ok, then this isn't a dict, so probably cleaner to invert-return
if !ok {
  return env.File, nil
}
for _, x := range args {